GoogleUtilities/Network/GULNetworkURLSession.m

```diff
@@ -387,28 +387,20 @@ - (void)URLSession:(NSURLSession *)session
         dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0);
 
     dispatch_async(evaluateBackgroundQueue, ^{
-      SecTrustResultType trustEval = kSecTrustResultInvalid;
       BOOL shouldAllow;
-      OSStatus trustError;
+      CFErrorRef errorRef = NULL;
 
       @synchronized([GULNetworkURLSession class]) {
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-        trustError = SecTrustEvaluate(serverTrust, &trustEval);
-#pragma clang diagnostic pop
+        shouldAllow = SecTrustEvaluateWithError(serverTrust, &errorRef);
```

Comment:
From [Apple docs](https://developer.apple.com/documentation/security/2980705-sectrustevaluatewitherror): _The SecTrustEvaluateWithError(_:_:) method returns a pass or fail indicator and an error describing the reason for any failure._